examples/aws-sagemaker/README.md

```diff
@@ -36,11 +36,11 @@ with `deepsparse` installed and creates an executable command `serve` that runs
 `invocations/` endpoint.
```

Comment:
This comment actually refers to line 34:
Change Double space in front of Sagemaker to a Single Space;

i.e from `on port 8080.  SageMaker will execute this image by running`
To: `on port 8080. SageMaker will execute this image by running`

Reply:
done!